spp_idpass/models/id_type.py

```diff
@@ -23,6 +27,12 @@ def unlink(self):
                 return super(OpenG2PIDType, self).unlink()
 
     def write(self, vals):
+        """
+        This overrides the write function to stop default ID
+        Types from being edited
+        :param vals: The Values being edited.
+        :return: raise an error if the ID Type being edited is id_type_idpass else write.
```

Comment:
Errors should not be in return, but in raises:
:raises: :class:`RuntimeError`: Out of fuel

Reply:
Done @jeremi